app/test_main.py

```diff
@@ -1,3 +1,57 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+def test_an_array() -> bool:
+    assert type(get_human_age(10, 3)) == list
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age",
+    [
+        (28, 28),
+        (-22, 20),
+        (22, 20),
+        (22, 120),
+        (22, 150),
+        (12, 180),
+        (-6, 60),
+        (-55, -120),
+        (-122, 150),
+    ],
+    ids=[
+        "test_main_get_human_age",
+        "test_main_get_human_age1",
+        "test_main_get_human_age2",
+        "test_main_get_human_age3",
+        "test_main_get_human_age4",
+        "test_main_get_human_age5",
+        "test_main_get_human_age6",
+        "test_main_get_human_age7",
+        "test_main_get_human_age8",
+    ]
+)
+def test_main_get(cat_age: int, dog_age: int) -> int:
+    result = get_human_age(cat_age, dog_age)
+    if cat_age == 28 and dog_age == 28:
```

Comment:
why did you use if station?? add expected result in @pytest.mark.parametrize. Please read this
https://docs.pytest.org/en/7.3.x/how-to/parametrize.html